libraries/ms-common/inc/gpio.h

```diff
@@ -74,3 +74,6 @@ StatusCode gpio_toggle_state(const GPIOAddress *address);
 
 // Gets the value of the input register for a pin and assigns it to the state that is passed in.
 StatusCode gpio_get_state(const GPIOAddress *address, GPIOState *input_state);
+
+// Test function used for setting the value of GPIO input registers (for X86 architechture)
+StatusCode test_gpio_set_input_state(const GPIOAddress *address, GPIOState state);
```

Comment:
This is unnecessary. Please remove it. It wouldn't work for STM32 anyways.

Reply:
ya this is before we talked. I'll mock it.